.github/workflows/pytest.yml

```diff
@@ -1,5 +1,5 @@
 name: pytest
-on: [push]
+on: [push, pull_request]
```

Comment:
@vhirtham so it is mandatory by now to specify the pr action ;)

Reply:
what did I miss? 😉 
just wanted to add this back in as quickly as possible 😃

Reply:
still curious why it differs between PRs ❓